docs/user/01-01-configuration.md

```diff
@@ -2,7 +2,7 @@
 title: Configuration
 ---
 
-The BTP Manager internal settings can be configured using CLI arguments or `ConfigMap`.
+You can configure the BTP Manager internal settings using CLI arguments or `ConfigMap`.
 
 To configure them using CLI arguments, follow this example:
```

Comment:
What does it mean `follow this example`? Should a user copy/paste it somewhere? Just read it?